### PR TITLE
add new sms codes from EQ 

### PR DESCRIPTION
**Pull request by LukeLoze1**
# Motivation and Context
add new sms codes from EQ 

# What has changed
New codes and tests

# How to test?
With other apps and ATs from ticket

# Links
https://trello.com/c/Ob0kC7WX/1359-new-sms-templates-and-sender-id-8

### CODE REVIEW

File: deactivate_uac_.csv [hunk 1]
```diff
@@ -0,0 +1,3 @@
+qid
```
Comment (NickGrantONS) [inline deactivate_uac_.csv:1]: what is this file and why's it been committed to this GitHub repo?

File: src/test/resources/definitions.json [hunk 2]
```diff
@@ -56,6 +56,7 @@
         "x-dead-letter-routing-key": "case.uac-qid-created"
       }
     },
+
```
Comment (NickGrantONS) [inline src/test/resources/definitions.json:59]: seems superfluous

**Review by NickGrantONS — CHANGES_REQUESTED** [review]
Minor changes

**Review by davidmort — APPROVED** [review]
All good once nicks comments have been fixed

**Review by NickGrantONS — APPROVED** [review]

**Pull request merged**